### PR TITLE
packages upgraded

### PR DESCRIPTION
**Pull request by deinok**

### CODE REVIEW

File: src/GraphQL/GraphQL.csproj [hunk 1]
```diff
@@ -30,14 +30,14 @@
...
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="System.Reactive.Core" Version="3.1.1" />
     <PackageReference Include="System.Reactive.Linq" Version="3.1.1" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.1'">
     <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.0.0" />
+    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.4.0" />
```
Comment (joemcbride) [inline src/GraphQL/GraphQL.csproj]: `4.4.0` lowest support is `netstandard1.3` so this needs to stay at `4.0.0`  The `netstandard.2.0` section targets `4.4.0`.

File: src/GraphQL.GraphiQL/packages.config [hunk 2]
```diff
@@ -21,8 +21,8 @@
   <package id="Microsoft.Owin.Security.OAuth" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Owin.Security.Twitter" version="3.0.1" targetFramework="net45" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
```
Comment (joemcbride) [inline src/GraphQL.GraphiQL/packages.config]: This should be kept at `net45`.

Comment (deinok) [reply src/GraphQL.GraphiQL/packages.config]: done

File: src/GraphQL/GraphQL.csproj (hunk 1 above)
Comment (deinok) [reply src/GraphQL/GraphQL.csproj]: done, test also not passing

**Comment by joemcbride** [comment]
Closes #443

**Pull request merged**